identity-and-access-management/secret-manager/concepts.mdx

```diff
@@ -62,7 +62,7 @@ Secrets are logical containers made up of zero or more immutable versions, that
 
 ## Secret Manager
 
-Secret Manager is a solution that provides a centralized location where your secrets can be stored and accessed. Its primary purpose is to securely store secrets, thus eliminating the need for developers to hard code sensitive information in applications or configuration files.
+Secret Manager is a solution that provides a centralized location where your secrets can be stored and accessed. Its primary purpose is to securely store secrets, thus eliminating the need for developers to hardcode sensitive information in applications or configuration files.
```

Comment:
Are you sure about this? According to the internet it is written as two words https://en.wiktionary.org/wiki/hard_code